src/Field/WfsSearch/WfsSearch.jsx

```diff
@@ -0,0 +1,389 @@
+     */
+    onSelect: (feature, olMap) => {
+      if (feature) {
+        const olView = olMap.getView();
+        const geoJsonFormat = new OlFormatGeoJSON();
+        const olFeature = geoJsonFormat.readFeature(feature);
+        const geometry = olFeature.getGeometry();
+
+        if (geometry) {
+          olView.fit(geometry, {
+            duration: 500
+          });
+        }
+      }
+    },
+    style: {
+      width: 200
+    }
+  }
+
+  /**
+   * Create the WfsSearch.
+   *
+   * @param {Object} props The initial props.
+   * @constructs WfsSearch
+   */
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchTerm: '',
+      data: []
+    };
+    this.onUpdateInput = this.onUpdateInput.bind(this);
+    this.onMenuItemSelected = this.onMenuItemSelected.bind(this);
+  }
+
+  /**
+   * Called if the input of the AutoComplete is being updated. It sets the
+   * current inputValue as searchTerm and starts a search if the inputValue has
+   * a length of at least `this.props.minChars` (default 3).
+   *
+   * @param {String|undefined} value The inputValue. Undefined if clear btn
+   *                                      is pressed.
+   */
+  onUpdateInput(value) {
+    this.setState({
+      data: []
+    });
+
+    if (value) {
+      this.setState({
+        searchTerm: value || ''
+      }, () => {
+        if (this.state.searchTerm.length >= this.props.minChars) {
+          this.doSearch();
+        }
+      });
+    }
+  }
+
+  /**
+   * Perform the search.
+   * @private
+   */
+  doSearch() {
+    const {
+      featureNS,
+      featurePrefix,
+      featureTypes,
+      geometryName,
+      maxFeatures,
+      outputFormat,
+      propertyNames,
+      srsName,
+      baseUrl,
+      additionalFetchOptions
+    } = this.props;
+
+    const options = {
+      featureNS,
+      featurePrefix,
+      featureTypes,
+      geometryName,
+      maxFeatures,
+      outputFormat,
+      propertyNames,
+      srsName,
+      filter: this.createFilter()
+    };
+
+    const wfsFormat = new OlFormatWFS();
+    const featureRequest = wfsFormat.writeGetFeature(options);
+    this.setState({fetching: true});
+
+    fetch(`${baseUrl}`, {
+      method: 'POST',
+      credentials: additionalFetchOptions.credentials
+        ? additionalFetchOptions.credentials
+        : 'same-origin',
+      body: new XMLSerializer().serializeToString(featureRequest),
+      ...additionalFetchOptions
+    })
+      .then(response => response.json())
+      .then(this.onFetchSuccess.bind(this))
+      .catch(this.onFetchError.bind(this));
+  }
+
+  /**
+   * Creates a filter fro the given searchAttributes prop and the current
+   * searchTerm.
+   * @private
+   */
+  createFilter() {
+    const {
+      searchTerm
+    } = this.state;
+    const {
+      searchAttributes
+    } = this.props;
+
+    const propertyFilters = searchAttributes.map(attribute =>
+      OlFormatFilter.like(attribute, `*${searchTerm}*`, '*', '.', '!', false));
```

Comment:
What happens here if numerical / boolean attributes are filtered / queried here?

Reply:
Uff i don't know. I think this depends on the WFS-Server. Either it throws an error or it just won't match... Anything we can do about it?

Reply:
for geoserver we can cast e.g. by using a string function like concatenate with an empty string, which will accept any input (according to the docs). this should be configurable defaulting to false probably.